gulpfile.js

```diff
@@ -1153,6 +1156,22 @@ var sandcastleJsHintOptions = ' + JSON.stringify(primary, null, 4) + ';';
     fs.writeFileSync(path.join('Apps', 'Sandcastle', 'jsHintOptions.js'), contents);
 }
 
+function buildSandcastle() {
+        return gulp.src([
+                'Apps/Sandcastle/**'
+            ])
+            // Replace require Source with pre-built Cesium
+            .pipe(gulpReplace('../../../ThirdParty/requirejs-2.1.20/require.js', '../../../CesiumUnminified/Cesium.js'))
+            // Use unminified cesium instead of source
+            .pipe(gulpReplace('Source/Cesium', 'CesiumUnminified'))
+            // Fix relative paths for new location
+            .pipe(gulpReplace('../../Source', '../../../Source'))
+            .pipe(gulpReplace('../../ThirdParty', '../../../ThirdParty'))
+            .pipe(gulpReplace('../../SampleData', '../../../../Apps/SampleData'))
+            .pipe(gulpReplace('Build/Documentation', 'Documentation'))
+            .pipe(gulp.dest('Build/Apps/Sandcastle'));
```

Comment:
I think all of the JPGs and PNGs are getting mangled here.

Reply:
Also, the root folder's `index.html` has a link to the built CesiumViewer, can we add a similar link to built Sandcastle?